### PR TITLE
hparams fixed for cnn-lstm

### PR DESCRIPTION
**Pull request by roshni-poddar**
configs.py: changed the mobilenet model to mobilenetv3_rw 
train_nn.py: fixed the hyperparameters for cnn+lstm

### CODE REVIEW

File: train_nn.py [hunk 1]
```diff
@@ -161,9 +161,7 @@ def fit(args):
             max_frame_len=169,
         )
         val_dataset = KeypointsDataset(
-            keypoints_dir=os.path.join(
-                args.data_dir, f"{args.dataset}_val_keypoints"
-            ),
+            keypoints_dir=os.path.join(args.data_dir, f"{args.dataset}_val_keypoints"),
```
Comment (grohith327) [inline train_nn.py:164]: Run Black

File: train_nn.py [hunk 2]
```diff
@@ -193,6 +191,9 @@ def fit(args):
         config = LstmConfig()
         if args.use_cnn:
             config.input_size = CnnConfig.output_dim
+            config.hidden_size = 128
+            config.num_layers = 2
+            config.dropout = 0
```
Comment (grohith327) [inline train_nn.py:196]: If this default config works the best for us, modify the config directly

File: train_nn.py [hunk 3]
```diff
@@ -224,14 +225,22 @@ def fit(args):
...
+            )
+        else:
+            scheduler.step(val_acc)
+            es(
+                model_path=model_path,
+                epoch_score=val_acc,
+                model=model,
+                optimizer=optimizer,
+                scheduler=scheduler,
+            )
```
Comment (grohith327) [inline train_nn.py:243]: Why not use a scheduler with cnn+lstm. I shouldn't give bad results, at the worst you should be getting the same result

**Review by grohith327 — CHANGES_REQUESTED** [review]
